odoo/service/server.py

```diff
@@ -117,6 +125,55 @@ def server_activate(self):
         if not self.reload_socket:
             super(ThreadedWSGIServerReloadable, self).server_activate()
 
+    def process_request(self, request, client_address):
+        """
+        Start a new thread to process the request.
+        Override the default method of class socketserver.ThreadingMixIn
+        to be able to get the thread object which is instantiated and maintain information about it.
+        """
+        t = threading.Thread(target = self.process_request_thread,
+                             args = (request, client_address))
+        t.daemon = self.daemon_threads
+        t.start()
+        self._request_threads[t] = {'start_time': time.time()}
+
+    def _handle_request_noblock(self):
+        """
+        In `socketserver`,
```

Comment:
What about this more simple text:
in the python socketserver process_request loop the __shutdown_request flag is not check between select and accept  thus when we set it to true a last request is accepted before exiting the loop. We override this function to add an additionalcheck before the accept()